File: src/zork/models/items/AxeAction.java
/*
 * To change this license header, choose License Headers in Project Properties.
 * To change this template file, choose Tools | Templates
 * and open the template in the editor.
 */
package zork.models.items;

import zork.interfaces.IUseable;

/**
 *
 * @author d.peters
 */
public class AxeAction implements IUseable {

    @Override
    public void use() {
        System.out.println("You swing the Axe at your enemy.");
    }
    
}

File: src/main/kotlin/ch/d/peters/zork/models/items/actions/AxeAction.kt
package ch.d.peters.zork.models.items.actions

import ch.d.peters.zork.interfaces.IItemAction

/**
 * Action for axes.
 *
 * @author Daniel Peters
 * @version 1.1.0
 */
 class AxeAction: IItemAction {
  /**
   * Use method without any param if the object should not do something with any object.
   */
  override fun execute () {
    System.out.println("You swing the Axe around you. You almost killed yourself in the process.")
  }
}
